ocs_ci/ocs/ui/views.py

```diff
@@ -939,6 +940,11 @@
         By.CSS_SELECTOR,
     ),
     "approve-storage-class-deletion": ("#confirm-action", By.CSS_SELECTOR),
+    "backing_store_type": (
+        "//*[text()='{}']/preceding-sibling::input[@name='backing-storage-radio-group']",
```

Comment:
Can we simply this Xpath?

Reply:
no, this calls axis. It bonds two elements: element with text and it's closest input.
We will use this type of locators more since Ui don't have id's for most of locators and all repeated html nodes created dynamically by JS